lib/sequenceserver/job.rb

```diff
@@ -122,6 +126,11 @@ def success!
       save
     end
 
+    def status!(status)
```

Comment:
Instead of creating a new method, I think we should modify `done!` to accept and save exit status.